app/components/modals/cropper-modal.js

```diff
@@ -2,15 +2,17 @@ import ModalBase from 'open-event-frontend/components/modals/modal-base';
 
 export default ModalBase.extend({
   onVisible() {
+    let viewPort = {};
+    let factor = 150;
+    const aspectRatio = this.getWithDefault('aspectRatio', [2, 1]);
+    viewPort.width = aspectRatio[0] * factor;
```

Comment:
Viewport is a [single word](https://en.oxforddictionaries.com/definition/viewport). So it should be `viewport` instead of `viewPort`